CONTRIBUTING.md

```diff
@@ -118,6 +119,43 @@ the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr)
 9. Open a PR in our repository (to the `develop` branch, **NOT** `main`) and
    follow the PR template so that we can efficiently review the changes.
 
+### 🧱 Pull Requests: Rebase Your Branch on Develop
+
+In case you have based your branch on `main` instead of `develop`, you can rebase your branch on `develop` by following the steps below:
+
+1. Make sure your branch is based on the `develop` branch. If it's not, you can create a new branch based on `develop` by running the following command:
+   ```
+   git checkout -b <new-branch-name> develop
+   ```
```

Comment:
If you already created your branch based on develop then there is no need to rebase right? So I think step 1 doesn't make sense here since if you have done this step then all the other steps are not necessary.

Reply:
Maybe you can move this information up, i.e.

```markdown
When making pull requests to ZenML, you should always make your changes on a branch that is based on `develop`. You can create a new branch based on `develop` by running the following command:
...

In case you have based your branch on `main` instead of `develop`, you can rebase your branch on `develop` by following the steps below:
...
```